nix-libs/nodeLib/tools/check-package-json

```diff
@@ -83,11 +83,19 @@ function checkDependencies() {
       return
     }
     console.error("OK (found version " + dependencyPackageObj.version + ")");
+
+    // If the package has any peer dependencies, check that they are satisfied
```

Comment:
Oh yeah, I'm just noticing this now. Nevermind my other more general comment about failing early, this code does that for the user. :+1: